internals/secrethub/run_test.go

```diff
@@ -72,6 +72,16 @@ func TestParseEnv(t *testing.T) {
 				},
 			},
 		},
+		"success with tabs": {
+			raw: "key\t=\tvalue",
+			expected: []envvar{
+				{
+					key:        "key",
+					value:      "value",
+					lineNumber: 1,
+				},
+			},
+		},
```

Comment:
Can you add a testcase for white space before the key and one for white space after the value?